src/passwordgenerator/PasswordGenerator.java

```diff
@@ -0,0 +1,36 @@
+package src.passwordgenerator;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class PasswordGenerator implements ActionListener {
+    JFrame jframe;
+
+    PasswordGenerator() {
+        jframe = new JFrame("Nimbus Look and Feel");
```

Comment:
I feel UI related changes should be moved to their individual folder.
We should follow Clean Architecture principle and keep `UI` separate from `Entity`.